packages/react-tree-grid/stories/LiveMeetingsList.stories.tsx

```diff
@@ -267,49 +243,45 @@ const MeetingsSectionItem = (props: MeetingsSectionItemProps) => {
       aria-description={`Created by: ${props.owner}. ${
         props.status ? `Meeting status: ${props.status}` : ''
       }`}
-      className={styles.sectionItem}
+      className={styles.treeGridRow}
     >
-      <TreeGridCell
-        aria-label={`${props.header}. ${props.location}`}
-        className={mergeClasses(styles.header, styles.container)}
-        header
-      >
-        <Avatar
-          className={styles.icon}
-          icon={<CalendarRegular />}
-          aria-hidden
-        />
-        <Button
-          appearance="transparent"
-          aria-label="Go to meeting"
-          className={mergeClasses(styles.noPadding, styles.title)}
-        >
+      <TreeGridCell className={styles.icon}>
+        <Avatar icon={<CalendarRegular />} aria-hidden />
+      </TreeGridCell>
+      <TreeGridCell header className={styles.title}>
+        <Button className={styles.noPadding} appearance="transparent">
           <Body1Stronger>{props.header}</Body1Stronger>
         </Button>
-        {props.status === 'missed' && (
-          <Tag
-            className={mergeClasses(styles.missedTag, styles.tag)}
-            size="small"
-            shape="circular"
-          >
-            <Caption1Stronger>Missed</Caption1Stronger>
-          </Tag>
-        )}
-        <Caption1 className={styles.location}>
+      </TreeGridCell>
+      <TreeGridCell className={styles.location}>
+        <Caption1>
           {props.location}, {props.owner}
         </Caption1>
-        {props.description && (
-          <Caption1 className={styles.description}>
-            {props.description}
-          </Caption1>
+      </TreeGridCell>
+      <TreeGridCell className={styles.description}>
+        {props.description && <Caption1>{props.description}</Caption1>}
+      </TreeGridCell>
+      <TreeGridCell className={styles.tag}>
+        {props.status === 'missed' && (
+          <Tag className={styles.missedTag} size="small" shape="circular">
+            <Caption1Stronger aria-label="status: missed">
+              Missed
+            </Caption1Stronger>
+          </Tag>
         )}
       </TreeGridCell>
-      {props.tasks && <TreeGridCell>{props.tasks}</TreeGridCell>}
-      {props.attachments && <TreeGridCell>{props.attachments}</TreeGridCell>}
-      <TreeGridCell>
+      {props.tasks && (
+        <TreeGridCell className={styles.tasks}>{props.tasks}</TreeGridCell>
+      )}
+      {props.attachments && (
+        <TreeGridCell className={styles.attachments}>
+          {props.attachments}
+        </TreeGridCell>
+      )}
```

Comment:
what is the conclusion on assymetric rows for treegrids? should we have them or not?

Reply:
There's no conclusion, we're focusing at the moment on the proper narration when you navigate on a nested level (from row to cells).

I believe the asymmetry might be an issue when navigating between rows, on a vertical level, we still need to verify that with more screen readers.

Reply:
Good point I think would be better to render just empty gridcell. 

Because when trying to navigation with NVDA in column navigation ctrl+alt+up/down arrows, then when there is asymmetry, then screen reader jump to different column automatically. 

This is good point how to solve it properly :)

Reply:
Ok, just updated this, now it should be symmetric in this example 👍🏻